src/components/Reports/FourteenMonthReports/FourteenMonthReport.tsx

```diff
@@ -42,7 +42,7 @@ export const FourteenMonthReport: React.FC<Props> = ({
   currencyType,
   isNavListOpen,
   title,
-  onSelectContact,
+  getContactUrl,
```

Comment:
Instead of prop-drilling `getContactUrl` down from the parent, what do you think about using the `useGetContactUrls` hook in this component and passing in the path based on the currency type? I can see a case for leaving it as-is, so let me know what you think.

Reply:
I would prefer to keep the controls in one place and since it's only going down 1 component, I think we are okay